### PR TITLE
DOCS Add docs for CMS help menu links

### PR DESCRIPTION
**Pull request by sachajudd**
Relies on https://github.com/silverstripe/silverstripe-admin/pull/615 to be merged first.

### CODE REVIEW

File: docs/en/02_Developer_Guides/15_Customising_the_Admin_Interface/How_Tos/Customise_CMS_Menu.md [hunk 1]
```diff
@@ -107,6 +107,19 @@ class. For more information about configuring extensions see the
 LeftAndMain::add_extension('CustomLeftAndMain')
 ```
 
+## Customising the CMS help menu
+
+The CMS help menu links in the south toolbar are configurable via your [configuration file](../../configuration).
+You can edit existing or add new links as shown in the example below:
+
+```yml
+// app/_config/config.yml
```
Comment (maxime-rainville) [inline docs/en/02_Developer_Guides/15_Customising_the_Admin_Interface/How_Tos/Customise_CMS_Menu.md]: That's not a valid YML comment ... replace // with #

File: docs/en/02_Developer_Guides/15_Customising_the_Admin_Interface/How_Tos/Customise_CMS_Menu.md [hunk 2]
```diff
@@ -107,6 +107,19 @@ class. For more information about configuring extensions see the
...
+SilverStripe\Admin\LeftAndMain:
+  help_links:
+    'CMS User help': 'my-custom-userhelp-docs.com'
```
Comment (maxime-rainville) [inline docs/en/02_Developer_Guides/15_Customising_the_Admin_Interface/How_Tos/Customise_CMS_Menu.md]: Can we use proper URLs, ideally to `https://example.com` or `https://example.org` or `https://example.net`.

Some out there will copy paste this into their code and not realised they need the protocol for this to work.

Can we add an example illustrating how to suppress a link we don't want.

File: docs/en/02_Developer_Guides/15_Customising_the_Admin_Interface/How_Tos/Customise_CMS_Menu.md [hunk 3]
```diff
@@ -107,6 +107,20 @@ class. For more information about configuring extensions see the
...
+The CMS help menu links in the south toolbar are configurable via your [configuration file](../../configuration).
+You can edit, add or remove existing links as shown in the examples below:
+
+```yml
+# app/_config/config.yml
+SilverStripe\Admin\LeftAndMain:
+  help_links:
+    'CMS User help': 'https://example.com'
+    'Additional link': 'https://example.org'
+    'Feedback': ''
```
[the remark this answers is not in the document]
Comment (sachajudd) [reply docs/en/02_Developer_Guides/15_Customising_the_Admin_Interface/How_Tos/Customise_CMS_Menu.md]: How's this @maxime-rainville ? 🙂

[the remark this answers is not in the document]
Comment (maxime-rainville) [reply docs/en/02_Developer_Guides/15_Customising_the_Admin_Interface/How_Tos/Customise_CMS_Menu.md]: My pedantic tendencies have reached a natural conclusion.

**Pull request merged**